ccp/apis/v1/ccp_server/swagger/swagger.yaml

```diff
@@ -0,0 +1,671 @@
+        desired_tag: "desired_tag"
+        build_status: "build_status"
+  TargetFile:
+    properties:
+      meta:
+        $ref: "#/definitions/Meta"
+      prebuild:
+        type: "boolean"
+      target_file_link:
+        type: "string"
+    title: "Link to target Dockerfile"
+    description: "Link to target Dockerfile for given project, also return boolean\
+      \ if project has prebuild defined"
+    example:
+      prebuild: true
+      meta:
+        apiVersion: "apiVersion"
+        timestamp: "timestamp"
+      target_file_link: "target_file_link"
+  ProjectBuildNameStatus:
+    properties:
+      build:
+        type: "string"
+      status:
+        type: "string"
+    title: "Build name and status"
+    description: "Build name and status"
+    example:
+      build: "build"
+      status: "status"
+  ProjectWeeklyScanBuildNameStatus:
+    properties:
+      weeklyscan-build:
+        type: "string"
+      status:
+        type: "string"
+    title: "Weekly scan build name and status"
+    description: "Weekly scan build name and status"
+    example:
+      weeklyscan-build: "weeklyscan-build"
+      status: "status"
+  ProjectBuilds:
+    properties:
+      build_number:
+        $ref: "#/definitions/ProjectBuildNameStatus"
+    example:
+      build_number:
+        build: "build"
+        status: "status"
+  ProjectWeeklyScanBuilds:
+    properties:
+      build_number:
+        $ref: "#/definitions/ProjectWeeklyScanBuildNameStatus"
+    example:
+      build_number:
+        weeklyscan-build: "weeklyscan-build"
+        status: "status"
+  ProjectBuildsInfo:
+    properties:
+      meta:
+        $ref: "#/definitions/Meta"
+      builds:
+        $ref: "#/definitions/ProjectBuilds"
+    title: "All the builds info for given project"
+    description: "All builds info for given project"
+    example:
+      meta:
+        apiVersion: "apiVersion"
+        timestamp: "timestamp"
+      builds:
+        build_number:
+          build: "build"
+          status: "status"
+  WeeklyScanBuildsInfo:
+    properties:
+      meta:
+        $ref: "#/definitions/Meta"
+      wscan-builds:
+        $ref: "#/definitions/ProjectWeeklyScanBuilds"
+    title: "All the weekly scan builds info for given project"
+    description: "All the weekly scan builds info for given project"
+    example:
+      wscan-builds:
+        build_number:
+          weeklyscan-build: "weeklyscan-build"
+          status: "status"
+      meta:
+        apiVersion: "apiVersion"
+        timestamp: "timestamp"
+  ScannerLogs:
+    properties:
+      logs:
+        type: "string"
+      description:
+        type: "string"
+    title: "Scanner log"
+    description: "All the scanner logs"
+    example:
+      description: "description"
+      logs: "logs"
+  AllScannerLogs:
+    properties:
+      scanner_name:
+        $ref: "#/definitions/ScannerLogs"
+    title: "All scanner logs"
+    description: "All scanner logs"
+    example:
+      scanner_name:
```

Comment:
Is the `scanner_name` going to be replaced with respective scanner names?

Reply:
yes, the word `scanner_name` won't make any sense anyway!!